vpr/src/base/read_blif.cpp

```diff
@@ -625,8 +700,19 @@ struct BlifAllocCallback : public blifparse::Callback {
      * to their nets
      */
     void merge_conn_nets() {
-        for (auto net_pair : curr_nets_to_merge_) {
-            curr_model().merge_nets(net_pair.first, net_pair.second);
+
+
+        // We need to check wether the driver_net was a previous sink. 
+        // if so, we need to remove the previous conn_pair. 
+       // trace_merge_conns_nets();  
+
+        if (getEchoEnabled()) {
+            dump_conn_netlist(getEchoFileName(E_ECHO_NETLIST_CONN)); 
+        }
+        merge_conn_tree();
```

Comment:
For consistent vertical spacing consider:
```suggestion
        }

        merge_conn_tree();
```